### PR TITLE
Blogapp: Add data models

### PR DESCRIPTION
**Pull request by mateo951**
- Added Add data models for users, posts, comments, and likes

### CODE REVIEW

File: db/migrate/20220219000729_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+    add_column :posts, :author_id, :bigint
+    add_foreign_key :posts, :users, column: :author_id
+    add_index :posts, :author_id
```
Comment (DammyShittu) [inline db/migrate/20220219000729_create_posts.rb:13]: [OPTIONAL]
- Awesome implementation of the foreign key. However, you'd agree that everyone loves a shorter way to do things. You do as well, right? 😉 

Here 👇 is a shorter way to set up a foreign key reference

```
t.references :author, foreign_key: { to_table: 'users' }
```

You can also apply the same implementation to the foreign keys in the other migration files.

**Review by DammyShittu — APPROVED** [review]
Hi  @mateo951, 

# APPROVED 🟢 

_Outstanding implementation of the features in this milestone ✔️_

Kudos to you 💯 

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

## CONGRATULATIONS! 🎉 

# OPTIONAL SUGGESTIONS ♻️ 

Kindly check the comment below 👇 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — COMMENTED** [review]
### Hi @mateo951 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keneogu — COMMENTED** [review]
> ## Hi @mateo951,
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> 
> ## Congratulations!     
> Cheers and Happy coding!clapclapclap
> 
> image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966) image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966)
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**